test/queue/QueueTest.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * The queue package contains our Queue, QueueTest, QueueNode and QueueTest 
+ * classes.
+ * 
+ * @author Scott McDonald and Jon Cronin
+ */
+package queue;
+
+import junit.framework.TestCase;
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.lang.Integer;
+
+/**
+ * Test class used to validate our simple Queue ADT.
+ * @author Scott McDonald and Jon Cronin
+ */
+public class QueueTest extends TestCase {
+    
+    // Default JUnit constructor created by Netbeans. Leaving for now.
```

Comment:
I tried using the automatic template that Netbeans creates as well for the stack implementation - I ended up modifying the code to mimic the test files we've already got.